src/settings/settings.cpp

```diff
@@ -458,6 +458,32 @@ SupportDistPriority SettingsBaseVirtual::getSettingAsSupportDistPriority(std::st
     return SupportDistPriority::XY_OVERRIDES_Z;
 }
 
+std::vector<int> SettingsBaseVirtual::getSettingAsIntegerList(std::string key) const
+{
+    std::vector<int> result;
+    std::string value_string = getSettingString(key);
+    if (!value_string.empty()) {
+        std::regex regex("([^,]+,?)");
```

Comment:
I think it does already because the regexp matches everything up to the next comma (and the comma) and then the matched string is passed to std::atoi() which copes with leading/trailing whitespace.